e2e/aws_test.go

```diff
@@ -0,0 +1,171 @@
+package e2e
+
+import (
+	. "github.com/onsi/ginkgo"
+	. "github.com/onsi/gomega"
+	corev1 "k8s.io/api/core/v1"
+	apierrors "k8s.io/apimachinery/pkg/api/errors"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+	awsv1 "sigs.k8s.io/cluster-api-provider-aws/api/v1beta1"
+	clusterv1 "sigs.k8s.io/cluster-api/api/v1beta1"
+	"sigs.k8s.io/controller-runtime/pkg/client"
+	yaml "sigs.k8s.io/yaml"
+
+	configv1 "github.com/openshift/api/config/v1"
+	mapiv1 "github.com/openshift/api/machine/v1beta1"
+	"github.com/openshift/cluster-capi-operator/e2e/framework"
+)
+
+const (
+	awsMachineTemplatenName = "aws-machine-template"
+)
+
+var _ = Describe("Cluster API AWS E2E", func() {
+	var awsMachineTemplate *awsv1.AWSMachineTemplate
+	var machineSet *clusterv1.MachineSet
+
+	BeforeEach(func() {
+		if platform != configv1.AWSPlatformType {
+			Skip("Skipping AWS E2E tests")
+		}
+
+		framework.CreateCoreCluster(cl, clusterName, "AWSCluster")
+		createAWSCluster(cl, getAWSMAPIProviderSpec(cl))
```

Comment:
this object doesnt cleanup, is it possible to move this out and call creation once?

Reply:
thx